### DISCUSSION TITLE
Chocolatey include zip within the package

### DISCUSSION THREAD

**Discussion by haiderbari** [goreleaser/goreleaser#3801] [Ideas, Issue Triage, and General discussions]
Hi I have recently started using Goreleaser to create chocolatey packages. I know it will not package the binary with chocolatey package. Instead it uses URL pointing to the git repo releases for windows binaries. Is it possible to add a new feature to the configuration to allow the window binaries to be package with the chocolatey package. So instead of url64bit have file64 with the zip already in the toolDir location. Thanks for the awesome tool.

**Comment by caarlos0** [comment]
I rather not right now. I don't think many people really use chocolatey now that winget is out... at least I particularly don't.

Also, winget is simpler to package/use. 

Anyway, closing for now, if someone disagrees let me know.
